src/client/components/AnvilSignatureFrame.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+class AnvilSignatureFrame extends React.Component {
```

Comment:
These should use the new react-ui eh?

Reply:
Or looks like it does now? Can just remove these components and all the related stuff?

Reply:
Yep the example uses the react-ui packages now. Was initially building the components here for testing/styling, but gonna remove these files now.